go.mod

```diff
@@ -262,6 +262,7 @@ require (
 	github.com/shirou/gopsutil v3.21.10+incompatible // indirect
 	github.com/shopspring/decimal v1.3.1 // indirect
 	github.com/sirupsen/logrus v1.8.1 // indirect
+	github.com/smartcontractkit/chainlink-solana v0.2.10-0.20220208192802-307c6fba76f0 // indirect
```

Comment:
Does this help? I would've thought it made things more complicated.

Reply:
Just trying it out, trying to fix the integration test